go.mod

```diff
@@ -8,17 +8,15 @@ require (
 	github.com/btcsuite/btcd v0.0.0-20190424234938-150379531efe
 	github.com/btcsuite/btcutil v0.0.0-20190207003914-4c204d697803
 	github.com/cespare/cp v1.1.1 // indirect
-	github.com/coreos/go-etcd v2.0.0+incompatible // indirect
 	github.com/davecgh/go-spew v1.1.1
 	github.com/deckarep/golang-set v1.7.1 // indirect
 	github.com/elastic/gosigar v0.10.5 // indirect
 	github.com/ethereum/go-ethereum v1.9.6
 	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5 // indirect
 	github.com/gballet/go-libpcsclite v0.0.0-20190607065134-2772fd86a8ff // indirect
-	github.com/go-delve/delve v1.4.1 // indirect
 	github.com/go-kit/kit v0.10.0
-	github.com/go-language-server/uri v0.2.0 // indirect
 	github.com/google/uuid v1.1.1
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
```

Comment:
Double check your dependencies, not sure why they're different.

Reply:
I used 'go mod tidy' and it ended up like this. Is 'go mod tidy' safe?

Reply:
Let me roll back the changes to play safe.